models/migrations/migrations.go

```diff
@@ -464,8 +464,6 @@ func dropTableColumns(sess *xorm.Session, tableName string, columnNames ...strin
 			sess.Rollback()
 			return fmt.Errorf("Drop table `%s` columns %v: %v", tableName, columnNames, err)
 		}
-
-		return sess.Commit()
```

Comment:
Is it safe to no longer commit here? It would affect all migrations, right?

Reply:
I have checked. This commit only occurs on the MSSQL branch and is the reason why this bug has occurred.

This is the problem with that sess.Commit() it gives the wrong impression.

Users of dropTableColumns have to commit their transactions at the end of the function.